letsencrypt/storage.py

```diff
@@ -50,10 +50,11 @@ def add_time_interval(base_time, interval, textparser=parsedatetime.Calendar()):
     return textparser.parseDT(interval, base_time, tzinfo=tzinfo)[0]
 
 
-def write_renewal_config(filename, target, relevant_data):
+def write_renewal_config(o_filename, n_filename, target, relevant_data):
```

Comment:
(The reason for specifying `o_filename` and `n_filename` is to avoid disturbing the use of `os.rename` in `update_configuration`, given that we are now creating a renewal config based on the old one rather than simply blindly writing the values from scratch.)